numpy/lib/index_tricks.py

```diff
@@ -68,6 +71,15 @@ def ix_(*args):
     array([[2, 4],
            [7, 9]])
 
+    >>> ixgrid = np.ix_([True,True], [2,4])
+    >>> a[ixgrid]
+    array([[2, 4],
+           [7, 9]])
+    >>> ixgrid = np.ix_([True,True], [False, False, True, False, True])
```

Comment:
I'd probably favor cramping it for consistency with the other examples in this section, or at least within the same line